core/client/fs/src/test/java/alluxio/client/file/MockFileSystemMasterClient.java

```diff
@@ -120,8 +121,9 @@ public List<URIStatus> listStatus(AlluxioURI path, ListStatusPOptions options)
   }
 
   @Override
-  public void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountPOptions options)
+  public MountPResponse mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountPOptions options)
       throws AlluxioStatusException {
+    return MountPResponse.getDefaultInstance();
```

Comment:
Make this return null to align with other method impls.

Reply:
done